### ISSUE TITLE
HDR breaks MicaForEveryone

### ISSUE THREAD

**Issue by hdmiport** [MicaForEveryone/MicaForEveryone#399]
When using HDR, text fails to render on windows where MicaForEveryone is applied.

**Comment by alesji** [comment]
Oh, so thats why - I was wondering why it doesn't work for me :D and it is extremely laggy, not sure if thats mica for everyone case in general or HDR has something to do with that too, but when i activate Mica for Everyone, moving windows and any work with them becomes incredibly laggy and slow.

**Comment by MuscularPuky** [comment]
> When using HDR, text fails to render on windows where MicaForEveryone is applied.

that's Windows bug. it hard to fix :(

**Comment by alexshhadeh** [comment]
Ping again, I just bought a HDR screen and would love for MicaForEveryone to work correctly on it.

**Comment by FreeAtorio** [comment]
This has annoyed me for a very long time will this ever get fixed? The only option is to just turn off HDR or turn off Mica/use a theme without Mica in Rectify11

**Comment by Sombrax25** [comment]
Yeah i ended up turning off HDR and only turning it on when i game.

**Comment by TheSubMatrix** [comment]
Can confirm this issue still exists and I had to uninstall it to make sure I could actually use the explorer on my monitor. I hope that a workaround or fix can eventually be found

**Comment by MuscularPuky** [comment]
> Can confirm this issue still exists and I had to uninstall it to make sure I could actually use the explorer on my monitor. I hope that a workaround or fix can eventually be found

GDI text rendering bug in HDR/WCG is Microsoft's limitation so we can't fix far now but other repository https://github.com/ramensoftware/windhawk-mods recently proofed a solution hooking and replacing GDI to Direct2D to fix the HDRtext bug (but that couldn't target in class for mica-ifying so both have pros and cons)
